ui/App/OrgScreen/OrgHeader.svelte

```diff
@@ -102,7 +102,7 @@
           </div>
         {/if}
       </div>
-      {#if websiteUrl || githubUrl || twitterUrl || seedId || seedApi}
+      {#if websiteUrl || githubUrl || twitterUrl || seedId || seedHost}
```

Comment:
```suggestion
      {#if websiteUrl || githubUrl || twitterUrl || (seedId && seedHost)}
```

We don't want the outer wrapper to show up in case only one of the seed attributes is present.